### DISCUSSION TITLE
EB support in Non-Uniform grid spacing

### DISCUSSION THREAD

**Discussion by ankithadas** [AMReX-Codes/amrex#3850] [Q&A] [answered]
Dear AMReX team,

Does EB functions such as `getBndryCent` work for non-uniform grids, i.e., $dx \neq dy$

Kind regards,
Ankith

**Comment by WeiqunZhang** [comment, marked as answer]
It works in some sense. In 2D, it should be all fine. But in 3d, the EB information will not be as accurate as it could be if dx != dy. This is because the calculation in 3D does not take the non-uniform spacing into account.
